Sources/Host/Linux/PTrace.cpp

```diff
@@ -255,19 +255,16 @@ ErrorCode PTrace::suspend(ProcessThreadId const &ptid) {
 
 ErrorCode PTrace::step(ProcessThreadId const &ptid, ProcessInfo const &pinfo,
                        int signal, Address const &address) {
+#if defined(ARCH_ARM)
+  DS2BUG("single stepping for ARM must be implemented in software");
+#endif
```

Comment:
I was thinking just hide the definition (to get a link error), not also the declaration (to get a compile error).